src/components/LeftMenu.tsx

```diff
@@ -85,13 +84,12 @@ export function LeftMenu() {
         </button>
       </header>
       <FileBrowser />
-      {isPhy() && (
-        <footer className={styles.leftMenuHeader}>
-          <button className={styles.iconButton} onClick={appContext.cdn.toggle}>
-            Upload to CDN
-          </button>
-        </footer>
-      )}
+      {/* Previously used by Physics only to upload files to CDN */}
```

Comment:
might as well just remove it - we can always go back to earlier versions in the repo if we need it